src/commands/dora/deployment.ts

```diff
@@ -0,0 +1,152 @@
+import {AxiosError} from 'axios'
+import chalk from 'chalk'
+import {Command, Option} from 'clipanion'
+import simpleGit from 'simple-git'
+import * as t from 'typanion'
+
+import {gitRepositoryURL, gitHash} from '../../helpers/git/get-git-data'
+import {Logger, LogLevel} from '../../helpers/logger'
+import {retryRequest} from '../../helpers/retry'
+
+import {apiConstructor} from './api'
+import {APIHelper, DeploymentEvent, GitInfo} from './interfaces'
+import {
+  renderDryRun,
+  renderFailedRequest,
+  renderGitWarning,
+  renderRequest,
+  renderRetriedRequest,
+  renderSuccessfulRequest,
+} from './renderer'
+
+const nonRetriableErrorCodes = [400, 403]
+
+export class SendDeploymentEvent extends Command {
+  public static paths = [['dora', 'deployment']]
+
+  public static usage = Command.Usage({
+    category: 'CI Visibility',
+    description: 'Send a new Deployment event for DORA Metrics to Datadog.',
+    details: `
+      This command will send details about a service Deployment to Datadog.\n
```

Comment:
is `\n` necessary?

Reply:
It's copied from the junit command. I think it looks a tiny bit better in the terminal if we leave the \n